tests/baselines/reference/privateNameField.errors.txt

```diff
@@ -0,0 +1,12 @@
+tests/cases/conformance/classes/members/privateNames/privateNameField.ts(2,5): error TS18006: Private names are not currently supported 
+
+
+==== tests/cases/conformance/classes/members/privateNames/privateNameField.ts (1 errors) ====
+    class A {
+        #name: string;
+        ~~~~~
+!!! error TS18006: Private names are not currently supported 
+        constructor(name: string) {
+            this.#name = name;

```

Comment:
Will this error if I don't have a property declaration but still try to access a private name?

```typescript
class A {
     func() { return this.#myValue; }
}
```

Reply:
Good question, apparently not. Not sure if this is intentional.

So the workflow seems to be:

- Fix any other parse errors, then try to recompile
- Then get the "Private names are not currently supported" error

Might not be ideal, will seek TS team feedback and suggestions.